File: searching and sorting algo/K-th Element of Two Sorted Arrays.cpp
int kthElement(int arr1[], int arr2[], int n, int m, int k){
        int count=0,flag;
        int i=0,j=0;
        for(;i<n && j<m && count!=k;j++,i++){
            if(arr1[i]<=arr2[j]){
                i++;
                count++;
                flag=0;
            }
            else if(arr1[i]>=arr2[j]){
                j++;
                count++;
                flag=1;
            }
            if(count ==k){
                if(flag)
                return arr2[j-1];
                else
                return arr1[i-1];
            }
        }
        while(i<n){
            i++;
            count++;
            if(count ==k){
                return arr1[i-1];
        }
            
        }
        while(j<n){
            j++;
            count++;
            if(count ==k){
                return arr2[j-1];
        }
    }
    return -1;
        }

File: searchingAndSorting/K-th_Element_of_Two_Sorted_Arrays.java
class Solution {
    public long kthElement( int arr1[], int arr2[], int n, int m, int k) {
        long[] arr=new long[n+m];
        for(int i=0;i<n;i++){
            arr[i]=arr1[i];
        }
        for(int i=0;i<m;i++){
            arr[i+n]=arr2[i];
        }
        Arrays.sort(arr);
        return arr[k-1]; 
    }
}
